package.json

```diff
@@ -246,6 +245,8 @@
     "test-integration-db2": "cross-env DIALECT=db2 yarn test-integration",
     "test-integration-ibmi": "cross-env DIALECT=ibmi yarn test-integration",
     "test-integration-snowflake": "cross-env DIALECT=snowflake yarn test-integration",
+    "test-integration-all": "yarn test-integration-mariadb && yarn test-integration-mysql && yarn test-integration-postgres && yarn test-integration-postgres-native && yarn test-integration-sqlite && yarn test-integration-mssql && yarn test-integration-db2 && yarn test-integration-ibmi && yarn test-integration-snowflake",
```

Comment:
Don't we want to make this `test-integration-oldest` and `test-integration-latest`?

Reply:
Same reason as above, because oldest & latest run on the same port, they can't run at the same time, so `test-integration-oldest` & `test-integration-latest` would do the same thing as `test-integration-all`